pyproject.toml

```diff
@@ -38,6 +38,7 @@ coverage = "^7.5.0"
 docker = "^7.0.0"
 pytest-md = "^0.2.0"
 pytest-emoji = "^0.2.0"
+regex = "^2024.5.10"
```

Comment:
Should be a non-dev dependency, since the omegaconf migration implementation depends on the library